docs/releaseNotes.yml

```diff
@@ -99,6 +99,20 @@ items:
           <code>*-migration.yaml</code> files have not been part of the migration instructions, and
           while the <code>*-agent.yaml</code> files remained part of the instructions they were
           actually unnescessary.
+      - title: Metric naming change for grpc_stats 
+        type: change
+        body: >-
+          The previous version of $productName$ was based on Envoy 1.17 and when using grpc_stats
```

Comment:
Maybe change this to `The previous versions of $productName$ were based...` since we had a lot more than one release built on Envoy 1.17